### DISCUSSION TITLE
Need API Endpoint for Retrieving All Customers and/or Searching for Customers

### DISCUSSION THREAD

**Discussion by nara-golfdistrict-com** [juspay/hyperswitch#3808] [Ideas / Feature Requests]
I just looked at the postman collection provided in the below document and could not find an API endpoint to retrieve all customer or search for a specific customer. Here are some usecases.
1. Check to see whether we created duplicate customers.
2. Migrate all customers to another platform.
https://api-reference.hyperswitch.io/introduction#welcome-to-hyperswitch-api-reference

Can you please add those endpoints?

**Comment by Narayanbhat166** [comment]
> API endpoint to retrieve all customer

[This endpoint](https://api-reference.hyperswitch.io/api-reference/customers/customers--list) can be used to retrieve list of customers. However it currently does not support filtering by `customer_id`. You can share any other use cases as well.

**Reply by nara-golfdistrict-com** [reply]
@Narayanbhat166 Thanks for your quick response. 
I think the above endpoint is enough. I didn't see that endpoint in the PostMan collection. Can you please get the postman collection updated?
<img width="305" alt="image" src="https://github.com/juspay/hyperswitch/assets/145275696/0be30806-edcf-47ed-982a-e30cbc670044">

**Comment by Narayanbhat166** [comment]
Hi @nara-golfdistrict-com, If you are using this postman collection, then the customers list endpoint is already present, you may want to pull the latest changes. https://www.postman.com/hyperswitch/workspace/hyperswitch/overview

**Reply by nara-golfdistrict-com** [reply]
Thanks @Narayanbhat166 for adding it to the postman collection.
